include/small/matras.h

```diff
@@ -157,7 +158,7 @@ struct matras_view {
 	/* root extent of the view */
 	void *root;
 	/* block count in the view */
-	matras_id_t block_count;
+	size_t block_count;
```

Comment:
Can't changing the type break [assertions](https://github.com/tarantool/tarantool/blob/1fcfb8c2bc8f3a4793c68cdc66930d8498c8504f/src/lib/salad/light.h#L949) in light? [`LIGHT(common)::table_size`](https://github.com/tarantool/tarantool/blob/1fcfb8c2bc8f3a4793c68cdc66930d8498c8504f/src/lib/salad/light.h#L142) still has type `uint32_t`. Are you planning to update it too?

Reply:
You're right, `count` and `table_size` in `LIGHT` also may overflow.
New decision: keep 32-bit `matras_view::block_count` and limit it to 2^31.